src/sagemaker/fw_utils.py

```diff
@@ -140,6 +149,18 @@ def _is_tf_14_or_later(framework, framework_version):
     )
 
 
+def _is_pt_12_or_later(framework, framework_version):
+    """
+    Args:
+        framework: Name of the frameowork
+        framework_version: framework version
+    """
+    asimov_lowest_pt = [1, 12, 0]
```

Comment:
Should gov region be handled here? We can do this in a separate PR.

Reply:
if the map above is filled out, it should work fine. not sure why an extra method is needed...

Reply:
lauren pointed this out, but this should be [1,2,0] and not [1,12,0]